cypress/e2e/network-lists.cy.js

```diff
@@ -0,0 +1,81 @@
+import generateUniqueName from '../support/utils';
+
+const selectors = {
+  sidebar: {
+    menuToggleButton: '[data-testid="sidebar-block__toggle-button"] > .p-button-icon',
+    networkListsMenuItem: '[data-testid="sidebar-block__menu-item__network-lists"] > .p-menuitem-text',
+    netowrkListScreenTitle: '.text-\\[var\\(--text-color\\)\\]'
+  },
+  networkLists: {
+    createButton: '[data-testid="create_Network List_button"] > .p-button-label',
+    nameInput: '[data-testid="network_list_name_input',
+    typeDropdown: '[data-testid="network_list_type_dropdown',
+    asnTextarea: '[data-testid="network_list_asn_list_textarea',
+    saveButton: '[data-testid="form-actions-submit-button"]',
+    createToastTitle: ':nth-child(2) > .p-toast-message-content > [data-testid="toast-block__toast"] > .flex > [data-testid="toast-block__message-summary-text"]',
+    createToastText: ':nth-child(2) > .p-toast-message-content > [data-testid="toast-block__toast"] > [data-testid="toast-block__message-detail-text"]',
+    createToastCloseButton: ':nth-child(2) > .p-toast-message-content > [data-pc-section="buttoncontainer"] > .p-toast-icon-close > .p-icon',
+    cancelButton: '[data-testid="form-actions-cancel-button"]',
+    searchInput: '[data-testid="data-table-search-input"]',
+    nameRow: '[data-testid="list-table-block__column__name__row"]',
+    typeRow: '[data-testid="list-table-block__column__listType__row"]',
+    actionButton: '[data-testid="data-table-actions-column-body-actions-menu-button"] > .p-button-icon',
+    deleteButton: '.p-menuitem-content > .p-menuitem-link > .p-menuitem-text',
+    deleteInput: '[data-testid="delete-dialog-confirmation-input-field"]',
+    confirmDeleteButton: '[data-testid="delete-dialog-footer-delete-button"] > .p-button-label',
+    deleteToastTitle: ':nth-child(3) > .p-toast-message-content > [data-testid="toast-block__toast"] > .flex > [data-testid="toast-block__message-summary-text"]',
+    deleteToastCloseButton: ':nth-child(3) > .p-toast-message-content > [data-pc-section="buttoncontainer"] > .p-toast-icon-close > .p-icon'
+  }
+}
+
+let networkListName = ''
+
+describe('Netowrk Lists spec', () => {
```

Comment:
I'll fix this typo in a next PR if no other problems are pointed out in this one: "Netowrk"